AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -60,6 +60,11 @@ - (instancetype)initWithNode:(ASDisplayNode *)node atIndex:(NSUInteger)index
   return self;
 }
 
+- (NSString *)description
+{
+  return [NSString stringWithFormat:@"<%@ %p node = %@, index = %ld>", self.class, self, _node, (long)_index];
```

Comment:
Let's go ahead and add the %p of the view and layer.  Their descriptions shouldn't be included since at least the view also includes the node's full description (IIRC)